src/app.js

```diff
@@ -3,10 +3,12 @@ import ReactDom from 'react-dom';
 import { BrowserRouter } from 'react-router-dom';
 import { Provider } from 'react-redux';
 import AppComponent from './AppComponent';
```

Comment:
there is no file `AppComponent.js` [here](https://github.com/OlgaLyakhvatskaya/homework/blob/saga/src/appComponent.js)